src/ray/core_worker/core_worker.cc

```diff
@@ -415,6 +421,13 @@ CoreWorker::CoreWorker(const CoreWorkerOptions &options, const WorkerID &worker_
                           : std::shared_ptr<LeasePolicyInterface>(
                                 std::make_shared<LocalLeasePolicy>(rpc_address_));
 
+  std::function<uint64_t()> get_max_pending_lease_requests_per_scheduling_category =
```

Comment:
nit: can have a private typedef for the function signature. Will improve readability.